docs/modules/ROOT/pages/servlet/authentication/architecture.adoc

```diff
@@ -2,7 +2,7 @@
 = Servlet Authentication Architecture
 :figures: servlet/authentication/architecture
 
-This discussion expands on xref:servlet/architecture.adoc#servlet-architecture[Servlet Security: The Big Picture] to describe the main architectural components of Spring Security's used in Servlet authentication.
+This discussion expands on xref:servlet/architecture.adoc#servlet-architecture[Servlet Security: The Big Picture] to describe the main architectural components of Spring Security used in Servlet authentication.
```

Comment:
I agree this could be clearer. What about:

```
... to describe the main architectural components that Spring Security uses in ...
```

Reply:
I agree. Done!